gatsby/src/components/looking-for-something/looking-for-something.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classes from './looking-for-something.module.scss';
+import { ArrowForwardIos, Phone } from '@material-ui/icons';
+
+const LookingForSomething = () => {
```

Comment:
could you expose all your fake texts like props

Reply:
Not sure how to approach

```html
<strong>Stále</strong> hledáte?
```

Two props? 🤔 Or render prop for passing already styled component?

Reply:
✅ exposed via props with default value - feel free to remove it in future.